distribution/docs/src/main/resources/content/_tables/MetacardAttributePlugin.adoc

```diff
@@ -18,17 +18,27 @@
 
 |Metacard Intersect Attributes:
 |intersectMetacardAttributes
-|String
-|Metacard attributes that will be collected and mapped to security information. Example: security.classification=classification Any duplicate destinations in this group will result in an intersection of values. E.g. source1=dest and source2=val where source1 is the set of
-|
-|true
+|List of rules
+|List of metacard attributes whose values will be collected (source attributes) and combined
+into a (new) metacard attribute (destination attribute). The format is the name of a source attribute,
+folowed by an equals sign, followed by a destination attribute. Intersection means that only the values
+common to all source attributes become values of the destination attribute.
+Note: duplicate destinations for Intersect and Union attributes behave unpredictably.
+
+Example If `codewords`=`compartments` is one rule and `sci-tokens=compartments` is another rule,
+then values common to **both** `codewords` and `sci-tokens` attributes will be added to a new
+attribute named `compartments`.
+|none
+|false
 
 |Metacard Union Attributes:
 |unionMetacardAttributes
-|String
-|'a', 'b', 'c'
-|null
-|true
+|List of rules
+|Similar to  `intersectMetacardAttributes`. Only the logic of how to combine the source values is different.
```

Comment:
❓ the docs don't match the metatype now, is that deliberate?

Reply:
That is an oversight.